sdk/communication/azure-communication-chat/azure/communication/chat/_generated/operations/_chat_operations.py

```diff
@@ -127,7 +126,7 @@ def list_chat_threads(
         start_time=None,  # type: Optional[datetime.datetime]
         **kwargs  # type: Any
     ):
-        # type: (...) -> Iterable["_models.ChatThreadsInfoCollection"]
+        # type: (...) -> Iterable["_models.ChatThreadsItemCollection"]
```

Comment:
I'm not sure if this is a bug in autorest or swagger spec - but I think this should be returning an `Iterable["_models.ChatThreadItem"]`.
It doesn't really matter in the generated code, so long as we've confirmed that the hand-written layer over the top is doing the right thing :)

Reply:
This is an autorest thing. It is exposed as `ItemPaged[ChatThreadItem]` to the outside world.